packages/react/src/SkipLink/SkipLink.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { SkipLink } from './SkipLink'
+import '@testing-library/jest-dom'
+
+describe('Skip link', () => {
+  it('renders', () => {
+    render(<SkipLink href="/" />)
+
```

Comment:
How about removing all empty lines from these simple and repetitive test functions?

Reply:
Hm, I don't know. I think the empty lines between the render, selection and test parts make it a bit more readable?